workers/unity/Packages/com.improbable.gdk.core/Systems/UpdateGroups.cs

```diff
@@ -13,18 +13,6 @@ public class InternalSpatialOSReceiveGroup
 
         [UpdateInGroup(typeof(SpatialOSReceiveGroup))]
         [UpdateAfter(typeof(InternalSpatialOSReceiveGroup))]
-        public class EntityInitialisationGroup
```

Comment:
What is the reason behind this change? Why not introduce a group rather than using
```
[UpdateBefore(typeof(SpatialOSReceiveGroup.GameObjectReceiveGroup))]
[UpdateAfter(typeof(SpatialOSReceiveGroup.InternalSpatialOSReceiveGroup))]
```
in `GameObjectInitializationSystem` et al?

Reply:
mainly because it's in playground and as we know: transitivity for unused groups doesn't work that well ;) we could introduce a group in the playground though. Can make that change

Reply:
That makes sense. Damn, this is really unfortunate.